packages/salesforcedx-vscode-apex-debugger/.vscodeignore

```diff
@@ -6,3 +6,22 @@ src/**
 **/*.map
 .gitignore
 tsconfig.json
+
+# salesforcedx-apex-debugger files to ignore 
```

Comment:
We're only using what's in `*/out/src/**` so I'm removing everything else from the vsix.

Reply:
Is it possible to ignore everything in the package and then whitelist `*/out/src/**` to cut down on the number of lines? Might be nice as we add more modules to the webpack config.

```
# ignore everything but out/src
node_modules/@salesforce/salesforcedx-apex-debugger/**
!node_modules/@salesforce/salesforcedx-apex-debugger/out/src/**
```

Reply:
Yes, this is only temporary and once we have webpack on every package we'll be able to ignore everything under the node_modules, test, etc. since everything will be bundled under the package's `out/src/**`. I did these updates so that we don't have to wait for every package to run via webpack in order to ship only the code needed to run.